lld/ELF/Config.h

```diff
@@ -331,6 +333,8 @@ struct Config {
   bool zText;
   bool zRetpolineplt;
   bool zWxneeded;
+  bool zForceZicfilp;
```

Comment:
I'm not sure if using `bool` here is suitable now. Currently, we have a second PLT scheme for Zicfilp: the function-signature-based one at riscv-non-isa/riscv-elf-psabi-doc#434 , so in the foreseeable future, this field would likely to be changed to an enum.